apps/whale-api/src/module.api/loan.controller.ts

```diff
@@ -233,6 +233,7 @@ export class LoanController {
       tokenId: detail.tokenId,
       token: mapTokenData(id, tokenInfo),
       factor: detail.factor.toFixed(),
+      activateAfterBlock: (detail as any)?.activateAfterBlock?.toNumber() ?? 0,
```

Comment:
```suggestion
      activateAfterBlock: 0,
```

Reply:
This exact same change was ported from `JellyfishSDK/whale` — because the tests were intact there didn't want to break them.

Reply:
Nvm, let's commit this.